presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorOutputTableHandle.java

```diff
@@ -60,6 +61,7 @@ public RaptorOutputTableHandle(
             @JsonProperty("distributionId") OptionalLong distributionId,
             @JsonProperty("bucketCount") OptionalInt bucketCount,
             @JsonProperty("organized") boolean organized,
```

Comment:
Reorder the params as: `bucketColumnHandles`, `organized`, and then `tableSupportsDeltaDelete` to match the stack order above.